tidypolars/tidypolars.py

```diff
@@ -144,6 +144,39 @@ def clone(self):
         """Very cheap deep clone"""
         return super().clone().pipe(from_polars)
 
+    def count(self, *args, sort: bool = False, name: str = 'N'):
```

Comment:
Can you make the default name `'n'` instead of `'N'`? It's `'n'` in dplyr. tidytable has it different because I built it with the data.table default name way at the beginning of tidytable and couldn't change it.